spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -672,6 +672,11 @@ public void setDefaultTopic(String defaultTopic) {
 		 */
 		private Long ackTime;
 
+		/**
+		 * True if this should be batch listener.
+		 */
+		private Boolean batchListener;
```

Comment:
I did not wan't to potentially change any existing behaviour and also saw that all other properties in ConcurrentKafkaListenerContainerFactoryConfigurer are handled as nullable so I just used the same approach here.

Thinking about it now, I don't see a reason why it shouldn't be set to 'false' - this is effectively the default behavior currently.